Git/commit.sublime-syntax

```diff
@@ -0,0 +1,81 @@
+%YAML 1.2
+---
+# Highlight regular git commits, merge commits, and tags.
+
+name: Git Commit
+file_extensions:
+  - COMMIT_EDITMSG
+  - MERGE_MSG
+  - TAG_EDITMSG
+scope: source.gitcommit
+
+contexts:
+  prototype:
+    - include: comments
```

Comment:
Including comments as prototype is useless as they must only start at the beginning of a line and must not be matched within a commit message line. Therefore the following construct is enough:

```
main:
    - include: comments
    - include: commit-message
```